libs/libvtrutil/src/vtr_assert.cpp

```diff
@@ -16,6 +16,8 @@ void handle_assert(const char* expr, const char* file, unsigned int line, const
         fprintf(stderr, " (%s)", msg);
     }
     fprintf(stderr, ".\n");
+    fflush(stdout);
+    fflush(stdin);
```

Comment:
What does it mean to flush the input stream?

Reply:
that have been fixed, I meant stdout and stderr.